RPerformanceTracking/Private/_RPTSender.m

```diff
@@ -204,7 +205,9 @@ - (void)writeMetric:(_RPTMetric *)metric
 
 - (void)writeMeasurement:(_RPTMeasurement *)measurement metricId:(NSString *)metricId
 {
-    if (measurement.endTime - measurement.startTime < MIN_TIME_MEASUREMENT) { return; }
+    _RPTTracker *tracker = _RPTTrackingManager.sharedInstance.tracker;
+    if ((!tracker.shouldTrackNonMetricMeasurements && !metricId.length) ||
```

Comment:
so this check will catch measurements that have already been added to the buffer before we get the response to decide enabling/disabling non-metric measurements?

Reply:
yes.